mh/quest_utils.py

```diff
@@ -0,0 +1,1564 @@
+import struct
```

Comment:
It's missing a licence header:
```python
#! /usr/bin/env python
# -*- coding: utf-8 -*-
"""Quest utils module.

    Monster Hunter 3 Server Project
    Copyright (C) 2022  NAME_TO_BE_CREDITED

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.

    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
"""
```
Don't forget to change `NAME_TO_BE_CREDITED`